### PR TITLE
CP-1883 Use null-aware operators

### PR DESCRIPTION
**Pull request by evanweible-wf**
_Fixes #144._

## Improvement
Now that we've bumped the minimum SDK version, we can leverage null-aware operators.

## Changes
Update all applicable areas in the example/source/test code where null-aware operators can be used to simplify null-checking logic.

## Testing
- [ ] CI passes

## Code Review
@trentgrover-wf 
@maxwellpeterson-wf 
@dustinlessard-wf 
@jayudey-wf 
@sebastianmalysa-wf

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

**Comment by rmconsole-wf** [comment]
@evanweible-wf
* It looks like you have not included any tickets in the title of this PR, but this repo requires tickets for all pull requests. Please create a ticket for this change, or click <a href='https://w-rmconsole.appspot.com/api/v1/pulls/85855641/addTicketToTitle'>here</a> to have Rosie create one for you.

File: lib/src/http/common/request.dart [hunk 1]
```diff
@@ -155,6 +155,7 @@ abstract class CommonRequest extends Object
   /// the [encoding].
   @override
   MediaType get contentType {
+    _contentType ??= defaultContentType;
```
Comment (trentgrover-wf) [inline lib/src/http/common/request.dart]: this bit doesn't look right...

Comment (evanweible-wf) [inline lib/src/http/common/request.dart]: good catch - I had started updating this method but realized the logic below wasn't as simple as I remembered.. I removed the rest of the changes but not this line

**Comment by evanweible-wf** [comment]
This has been rebased against latest 3.0.0-wip now that #220 has been merged. Also @trentgrover-wf I addressed your comment.

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - passing CI
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into WIP of 3.0.0.

**Comment by jayudey-wf** [comment]
@evanweible-wf should we bump the sdk version in the `pubspec.yaml`

**Comment by evanweible-wf** [comment]
@jayudey-wf 👊 that was supposed to be updated here: https://github.com/Workiva/w_transport/pull/197

No idea how I missed that, I'll fix it.

**Comment by evanweible-wf** [comment]
@trentgrover-wf @maxwellpeterson-wf @jayudey-wf updated the minimum SDK version in the latest commit.

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1

**Comment by maxwellpeterson-wf** [comment]
@jayudey-wf ready for you

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - Passing CI
- [ ] Unit test created/updated
- [ ] All unit tests pass 

Merging into master.

**Comment by evanweible-wf** [comment]
@rmconsole-wf run_merge_script

**Pull request merged**